src/components/scrollToTop/ScrollToTop.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react';
+import { Fab } from '@material-ui/core';
+import ArrowUpIcon from '@material-ui/icons/ArrowUpward';
+import "./ScrollToTop.scss";
+
+const ScrollArrow = () =>{
+
+  const [showScroll, setShowScroll] = useState(false);
+
+  const checkScrollTop = () => {
+    (window.pageYOffset > 500) ? setShowScroll(true) : setShowScroll(false)
+  };
+    
+  const scrollToTop = () =>{
+    window.scrollTo({top: 0, behavior: 'smooth'});
+  };
+
+  window.addEventListener('scroll', checkScrollTop);
```

Comment:
Can we put the addEventListener into a useEffect? here are two examples:

https://gist.github.com/JoaoTMDias/b5aeb6e02be7e6d25d5fd7f10826fc8a

https://stackoverflow.com/a/58833641/9830506

Reply:
yea , I will use useEffect to do that.